libsigopt/compute/misc/multimetric.py

```diff
@@ -18,30 +19,40 @@
 MULTIMETRIC_INITIALIZATION = "initialization"
 
 
-@dataclass(frozen=True, slots=True)
-class MultimetricInfo:
-  method: str
-  params: object
-
-
 @dataclass(frozen=True, slots=True)
 class ConvexCombinationParams:
   weights: numpy.ndarray
 
 
-@dataclass(frozen=True, slots=True)
-class OptimizeOneMetricParams:
+@dataclass(frozen=True, kw_only=True)
```

Comment:
did you mean to remove `slots` too?

Reply:
Actually yes, since this is inherited by the dataclasses below we need to remove the `slots` argument. Otherwise the inheritance raises `TypeError: multiple bases have instance lay-out conflict`

Reply:
cool, just checking